m4_sessions/1804inning/public_data.md

```diff
@@ -5,12 +5,56 @@ Anna Royer
 Sabrina Kennedy
 Jared Eklin
 
-Name of Session: 
+#### Name of Session: Using Public Data for Civic Tech
 
-Session Description (Why should students come to this session):
+#### Session Description (Why should students come to this session):
+Discussing and Engaging with material and others on what civic tech and public data are and how to use them to solve problems irl.
 
-Session Outcomes (What will students learn/leave with?):
+#### Session Outcomes (What will students learn/leave with?):
+* Students will be emboldened to become involved in Civic Tech projects
+* Students will know what Civic Tech is and how it is being used or has been used in the past
+* Students will have concrete resources to becoming involved in Civic Tech
+* Students will brainstorm ideas of ways to become involved in Civic Tech
+ 
+#### Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+**Ice Breakers** (7 minutes)
+TBD
 
-Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+**Intro** (10 minutes)
+* What is Civic Tech?
```

Comment:
Slides?  Key talking points?  Think about how to break this up and have each facilitator take part of it.